src/DefifaScoreCardVerifier.sol

```diff
@@ -0,0 +1,64 @@
+pragma solidity ^0.8.15;
+
+import "murky/Merkle.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./interfaces/IDefifaScoreCardVerifier.sol";
+
+
+/**
+Custom Errors
+*/
+error INVALID_SCORECARD();
+
+// Transfer ownership to governer on deploymment.
+contract DefifaScoreCardVerifier is IDefifaScoreCardVerifier, Merkle, Ownable {
```

Comment:
Do we need `Ownable` in this contract? since it has no state variables

Reply:
I was trying to maintain a pattern that only governer can call this,  but yeah ur right we don't need will remove that